doc/Architecture-overview/index.md

```diff
@@ -10,156 +10,157 @@ sort_idx: 15
 
-Endpoints may also be associated with [*owners*]({{root_url}}Programming-guide/Key-platform-features/Endpoint-ownership/).
-Depending on the application, owners may be persons, groups of people, or organizations.
+Endpoints can also be associated with [owners]({{root_url}}Programming-guide/Key-platform-features/Endpoint-ownership/).
+Depending on the application, owners can be persons, groups of people, or organizations.
 
-# Further reading
+## Further reading
 
-Use the following guides and references to make the most of Kaa.
+Use the following guides and references to learn more about Kaa features.
 
 | Guide                                                          | What it is for                                                                                                                                                                                           |
 |----------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Key system features]({{root_url}}Programming-guide/Key-system-features/)     | Use this reference to learn about features and capabilities of Kaa \([Endpoint profiling]({{root_url}}Programming-guide/Key-platform-features/Endpoint-profiles/), [Data collection]({{root_url}}Programming-guide/Key-platform-features/Data-collection/), [Configuration management]({{root_url}}Programming-guide/Key-platform-features/Configuration-management/), [Events]({{root_url}}Programming-guide/Key-platform-features/Events/), [Notifications]({{root_url}}Programming-guide/Key-platform-features/Notifications/), and other features\). |
-| [Installation guide]({{root_url}}Administration-guide/System-installation)    | Use this guide to install and configure Kaa either on a single Linux node or in a cluster environment.                                                                                                    |
-| [Contribute To Kaa]({{root_url}}Customization-guide/How-to-contribute/)       | Use this guide to learn how to contribute to Kaa project and which code/documentation style conventions we adhere to.                                                                                                   |
\ No newline at end of file
+| **[Key system features]({{root_url}}Programming-guide/Key-system-features/)**     | Learn about Kaa key features, such as [endpoint profiling]({{root_url}}Programming-guide/Key-platform-features/Endpoint-profiles/), [data collection]({{root_url}}Programming-guide/Key-platform-features/Data-collection/), [configuration management]({{root_url}}Programming-guide/Key-platform-features/Configuration-management/), [events]({{root_url}}Programming-guide/Key-platform-features/Events/), [notifications]({{root_url}}Programming-guide/Key-platform-features/Notifications/), and others. |
```

Comment:
**[Key system features]({{root_url}}Programming-guide/Key-system-features/)** -> Page not found. Please replace with **[Key platform features]({{root_url}}Programming-guide/Key-platform-features/)**